docs/source/installation.rst

```diff
@@ -0,0 +1,7 @@
+.. _installation:
+
+************
+Installation
+************
+
+Here is the documentation about the installation.
```

Comment:
I think this can be kept very short - `pip install rabbitmq` will almost do.
You may want to link to some existing instructions on how to install rabbitmq on the web (I guess there will be official ones)?

Reply:
The 'rmq' might be better to included in install_requires rather than as an extra requires?
https://github.com/aiidateam/kiwipy/pull/33